### PR TITLE
Dev/deez

### PR DESCRIPTION
**Pull request by IkkyOdufade**
1. added deezer as a music service.
2. added a login alert for users.

### CODE REVIEW

**Comment by IkkyOdufade** [comment]
This is the new branch with just deezer files. Kindly find the new corrections. Thank you for your understanding 👍 .

**Comment by IkkyOdufade** [comment]
@ianb

File: extension/services/deezer/player.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          if (thenPlay) {
+            const playerButton = await this.waitForSelector(SEARCH_PLAY, {
+              timeout: 2000,
+              // There seem to be 3 fixed buttons that appear early before the search results
+              minCount: 4,
+            });
+            playerButton.click();
+          }
+        } catch (e) {
+         const unlogged = this.querySelector("div[class='unlogged-homepage']")
```
Comment (ianb) [inline extension/services/deezer/player.js]: Formatting problem here

File: extension/services/deezer/player.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        }
+      }
+  
+      action_pause() {
+        const group_btn = this.querySelector("button[aria-label='Pause']");
+        group_btn.click();
+      }
+  
+      action_unpause() {
+        const group_btn = this.querySelector("button[aria-label='Play']");
```
Comment (ianb) [inline extension/services/deezer/player.js]: I'd probably just call all these `button`. In JavaScript generally mixedCase is preferred.

File: extension/services/deezer/player.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        group_btn.click();
+      }
+  
+      action_move({ direction }) {
+        let group_btn;
+        if (direction === "Next") {
+          group_btn = "button[aria-label='Next']";
+        } else if (direction === "Previous") {
+          group_btn = "button[aria-label='Previous']";
+        }
```
Comment (ianb) [inline extension/services/deezer/player.js]: This doesn't do anything with the button.

**Comment by IkkyOdufade** [comment]
Hi Ian, I made the changes you requested for. Looking forward to your feedback.

**Comment by IkkyOdufade** [comment]
Ok, I wanted to ask you about that, the tests I mean.

**Comment by IkkyOdufade** [comment]
Thank you.

**Comment by ianb** [comment]
Alright, looking good. We have Deezer support, yay! ;)

**Comment by IkkyOdufade** [comment]
Thank you @ianb . Your mentorship has been invaluable. My second commit to mozilla.. :)

**Pull request merged**